packages/hydration/vue.ts

```diff
@@ -17,7 +15,7 @@ export default function createVueIsland (component: Component, id: string, el: E
     appDefinition.name = `Island: ${nameFromFile(component.__file)}`
 
   const app = createVueApp(appDefinition)
-  app.mount(el!, Boolean(slots))
+  app.mount(el!, true)
```

Comment:
Not quite sure what the hydration setting has to do with slots. And as far as i could see `Boolean(slots)` was always `true` anyway. In my (few) tests, `slots` was either `{}` or `{foo:'bar'}` which both are truthy.

Reply:
Probably outdated code from one of the first versions.

I've [removed the second parameter this in this branch](https://github.com/ElMassimo/iles/blob/improve/vue-hydration/packages/hydration/vue.ts#L17) since `createSSRApp` will override `mount` to always pass `true`.